metrics/details/init.lua

```diff
@@ -99,6 +99,21 @@ end
 
 global_metrics_registry = Registry.new()
 
+------------------------------- Global Labels --------------------------------
+global_labels = {}
```

Comment:
Personally I don't like the idea of global variables. Moreover "global_labels" is quite common name. I can suppose that some user could redefine "global_labels" by mistake

Reply:
Move `global_labels` to registry as `global_metrics_registry.label_pairs`, add `Registry:set_labels(label_pairs)` method. Also, `metrics.clear()` call cleans global labels now.